__tests__/lib/tree-utils.test.ts

```
@@ -0,0 +1,314 @@
+        _id: "parent-node-2",
+        type: "folder",
+        name: "Parent Node 2",
+        pointer_id: "parent-node-ptr-2",
+        tenantId: "tenant-1",
+        createdAt: new Date().toISOString(),
+        updatedAt: new Date().toISOString(),
+        collaborative: false,
+      };
+      const children = [];
+      const children2 = [];
+      for (let i = 2; i >= -0; i--) {
```

Biome (2.1.2)

[error] Do not use the >= operator to compare against -0.

Safe fix: Replace -0 with 0


(lint/suspicious/noCompareNegZero)